components/icon-button/index.js

```diff
@@ -25,14 +25,21 @@ class IconButton extends Component {
 		const classes = classnames( 'components-icon-button', className );
 		const tooltipText = tooltip || label;
 
-		// Should show the tooltip if an explicit tooltip is passed
-		// or if there's a label and the children are empty and the tooltip is not explicitely disabled
-		const showTooltip = !! tooltip ||
+		// Should show the tooltip if...
+		const showTooltip = (
+			// an explicit tooltip is passed or...
+			tooltip ||
+			// there's a shortcut or...
```

Comment:
I think I mentioned elsewhere, but conceptually shortcut has nothing to do with `IconButton` and I wish it wasn't included here.